### PR TITLE
feat: proposal details skeleton, spacing, json color and url

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Various small improvements to polish the new proposal detail page ui.

# PR

- [x] require `gix-components` PR https://github.com/dfinity/gix-components/pull/52

# Changes

- display proposal `url`
- use new cmp `SkeletonText`
- update `json` color for new design
- adjust spacing to avoid layout big gap when user have lots of neurons
- wrap payload text

# Screenshots

Was:

Capture d’écran 2022-09-05 à 07 48 55
Capture d’écran 2022-09-05 à 07 49 04

Will be:

Capture d’écran 2022-09-05 à 07 47 10
Capture d’écran 2022-09-05 à 07 47 29

### CODE REVIEW

File: frontend/src/lib/components/proposal-detail/ProposalProposerInfoSection.svelte [hunk 1]
```diff
@@ -19,7 +20,13 @@
 
 <div class="content-cell-details">
   <ProposalSummary {proposal}>
-    <svelte:fragment slot="title">{title ?? ""}</svelte:fragment>
+    <svelte:fragment slot="title">
+      <h1>{title ?? ""}</h1>
+
+      {#if url}
+        <a target="_blank" href={url}>{url}</a>
```
Comment (mstrasinskis) [inline frontend/src/lib/components/proposal-detail/ProposalProposerInfoSection.svelte:27]: Do we need `rel="noopener noreferrer"` here?

**Review by mstrasinskis — APPROVED** [review]
LGTM, thanks!

File: frontend/src/lib/components/proposal-detail/ProposalProposerInfoSection.svelte (hunk 1 above)
Comment (peterpeterparker) [reply frontend/src/lib/components/proposal-detail/ProposalProposerInfoSection.svelte:27]: Yes. I copy/pasted it from another component. Will add it there too!

Comment (peterpeterparker) [reply frontend/src/lib/components/proposal-detail/ProposalProposerInfoSection.svelte]: Done, thanks!

**Pull request merged**